2017/firstEmailAnnouncement.txt

```diff
@@ -8,14 +8,17 @@ What is BarCampGR?
 
 What's the format?
-Presentations are 25 minutes long, with five minutes in between. You're welcome to sit in on a presentation or hang around in the lounge and talk; it's all the same with us. If you run a presentation, we don't care how you run it - so long as you don't get us in trouble! Give a monologue, a Q&A or a round table; it's your topic, your presentation, your audience. If people didn't want to spend 25 minutes on your topic, they wouldn't be there. Please be considerate when signing up for multiple talks at the start of the conference. If you see slots lingering open later on, feel free to fill them in then. This allows for more room for new talks and late adds, while also keeping you from accidentally scheduling an hour for a 25 minute talk!
+Presentations are 25 minutes long, with five minutes in between. You're welcome to sit in on a presentation or hang around in the lounge and talk; it's all the same with us. If you run a presentation, we don't care how you run it - so long as you don't get us in trouble! Give a monologue, a Q&A or a round table; it's your topic, your presentation, your audience. If people didn't want to spend 25 minutes on your topic, they wouldn't be there. 
 
 Don't know what to talk about?
 Certainly you have some relatively unique experiences. What do you do for a hobby? What's something you managed to fix that you're proud of? It doesn't matter if the height of your accomplishment is changing your car's oil or if you've war-driven half of the Grand Rapids area; if people are interested, they will show up at your talk. If they aren’t interested, they will probably attend one of the other talks during that time.
 
 But I'm not an expert!
 Sure you are! If you know the first thing about a subject, you know more than people who don't. And, yes, people who know more than you will probably attend your presentation. Interact with them; you both have something to learn from each other. Chances are, if you're both really interested in a subject, you'll find yourselves looking for each other in the lounge later.
 
+How do I register a talk?
+There's a whiteboard near the entrance; that's the schedule grid. On it, there's a list of rooms and times, and the grid starts off empty. Simply walk up to that board, find a spot, write in your topic and your name, and then show up in that room at that time to speak. It's that simple. Please be considerate when signing up for multiple talks at the start of the conference. If you see slots lingering open later on, feel free to fill them in then. This allows for more room for new talks and late adds, while also keeping you from accidentally scheduling an hour for a 25 minute talk!
```

Comment:
Some years, it's been a dry-erase board. Other years, it's been big shits of paper. Some years, it's been a bunch of small pieces of paper taped together or taped to the wall. Could be anything, really.

As far as the use of whiteboard as a generic term, that's pretty common already; people talk about whiteboarding conversations without caring about the physical medium used. You have virtual whiteboards in telecommunications and virtual meeting tools. So I don't see it as a confusing term.

s/whiteboard/big board/ if concerned.